src/Interfaces.test.ts

```diff
@@ -31,6 +43,24 @@ describe('Interfaces', async (should: any) => {
     })
   }
 
+  {
+    assert({
+      given: 'a valid Identity claim, isClaim',
+      should: 'return true',
+      actual: isClaim(Me),
+      expected: true,
+    })
+  }
+
+  {
+    assert({
+      given: 'a valid Identity claim, isIdentity',
+      should: 'return true',
+      actual: isIdentity(Me),
+      expected: true,
```

Comment:
Sure

Reply:
Because `isIdentity` takes a `Claim` type, you can not submit a non-Claim object. `isClaim` validates the general nature of the claim and runs separately from `isIdentity` or `isWork`, which only check the ClaimType. So I added a failure test for `isClaim` and a failure test for `isIdentity` using a different claim type.

Reply:
That works. You could also cast to Claim to get around this issue.

```ts
const myClaim = {} as Claim
```
That is just FYI for next time you come across the issue, this is good for now.